cmd/kyverno/main.go

```diff
@@ -27,12 +27,12 @@ import (
 var (
 	kubeconfig     string
 	serverIP       string
-	cpu            bool
-	memory         bool
```

Comment:
Didn't notice we removed the profiling flags, we should remove flags from install.yaml if it's deleted.
https://github.com/nirmata/kyverno/blob/ddbe15c6df38a0965bc05f1caa4261853f2f1c43/definitions/install.yaml#L545-L546